### PR TITLE
Milestone 3 - Contact Form & Footer Sections

### PR DESCRIPTION
**Pull request by chelmerrox**
Hi! This PR is for milestone 3. I'm using template 5 from Figma as my design guide. 
I have also completed its requirements to:

- add a form that has HTML validations
- use a Formspree service for the form's action attribute
- have pseudoclasses for the form's submit button
- stick to the styling of the template I chose

Thanks in advance for your review, comments & help!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -216,6 +216,61 @@ <h3 class="poppins-font font-white">Skills</h3>
...
+
+      <form action="https://formspree.io/f/xwkyyayl" method="POST" name="contact-form" class="contact-form" id="contact-form">
+        <div class="user-name-section" id="user-name-section">
+          <label for="user-name" class="poppins-font">Your name</label>
+          <input type="text" class="poppins-font user-name" id="user-name" name="user-name" placeholder="Anastasiia" maxlength="30" required/>
+        </div>
+        
+        <div class="user-email-section" id="user-email-section">
+          <label for="user-email" class="poppins-font">Your email</label>
+          <input type="text" class="poppins-font user-email" id="user-email" name="user-email" placeholder="anastassia@gmail.com" required/>
```
Comment (Lordkaito) [inline index.html:239]: - [ ] For the email input. please make sure that its type is equal to `email` and not `text`, so you will end up with `input type='email'` instead.

File: index.html [hunk 2]
```diff
@@ -216,6 +216,61 @@ <h3 class="poppins-font font-white">Skills</h3>
...
+        </div>
+
+        <textarea class="poppins-font" id="user-comments" name="user-comments" placeholder="Enter your message here..." maxlength="500"></textarea>
```
Comment (Lordkaito) [inline index.html:242]: - [ ] Please make sure to add the `required` property to your textarea, so your user cannot submit a form without a text.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @chelmerrox, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @chelmerrox,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**